From afbf95a478b6e94f532ca0776c79da1b08be7eed Mon Sep 17 00:00:00 2001
From: iglocska <andras.iklody@gmail.com>
Date: Tue, 19 Jan 2021 14:01:36 +0100
Subject: [PATCH] fix: [security] Require password confirmations by default

- the setting is optional, but the default should be that it's required unless disabled

- As reported by Patrix Kontura from ESET
---
 app/Config/config.default.php | 1 +
 1 file changed, 1 insertion(+)

diff --git a/app/Config/config.default.php b/app/Config/config.default.php
index c1f944dba7..ab36884274 100644
--- a/app/Config/config.default.php
+++ b/app/Config/config.default.php
@@ -6,6 +6,7 @@
 			'level'      => 'medium',
 			'salt'       => '',
 			'cipherSeed' => '',
+            'require_password_confirmation' => true
 			//'auth'=>array('CertAuth.Certificate'), // additional authentication methods
 			//'auth'=>array('ShibbAuth.ApacheShibb'),
 		),
